config/openshift/v4_9_exp/translate.go

```diff
@@ -272,5 +272,13 @@ func validateMCOSupport(mc result.MachineConfig, ts translate.TranslationSet) re
 			r.AddOnError(path.New("json", "spec", "config", "passwd", "users", i), common.ErrUserNameSupport)
 		}
 	}
+	for i := range mc.Spec.Config.KernelArguments.ShouldExist {
+		// IMMUTABLE
```

Comment:
IMMUTABLE requires that the MCO notices changes to the field and degrades the node, but it doesn't look as though https://github.com/openshift/machine-config-operator/pull/2613 currently does that.